### ISSUE TITLE
Color syntaxes for @if helper does not work in ST3

### ISSUE THREAD

**Issue by kurdin** [zanuka/dust-buster#38]
``` html
{@if cond="foo > 0"}
 <li id="lodgingLink" class="">
  <a href=""><i class="icon-lodging"></i></a>
 </li>
{/if}
```

It will brake all color syntaxes after that code in Sublime Text 3 .dust templates. 
This is because symbol `>` in `cond` does not handled well.

**Comment by zanuka** [comment]
good catch @kurdin , will fix... also keep in mind that @if has been deprecated:
https://github.com/linkedin/dustjs-helpers/wiki/Deprecated-Features

**Comment by zanuka** [comment]
fixed, v1.1.10 release tagged. package control should pick up the change within the hour: https://github.com/zanuka/dust-buster/tree/v1.1.10

**Comment by kurdin** [comment]
Thanks! Confirmed, it works.
